### PR TITLE
CLOUDP-112242: Updated README.MD

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-112242

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

This PR updates the `README.MD` to include atlascli. The new `README` is available here https://github.com/mongodb/mongocli/tree/CLOUDP-112242. 
**This PR will be merged once we have pre-released atlascli and renamed the repository to mongodb-atlas-cli**

**Next Step:**
- Pre-releasing `atlascli v0.0.1`
- Renaming the github repository to `mongodb-atlas-cli`
- Merging this PR
- Releasing `atlascli v1.0.0`
- Updating again the `atlascli.md` to add documentation links

### CODE REVIEW

File: README.md
Comment (blva) [inline README.md:121]: [q] once we rename the repo, this link will display mongo and atlascli releases in the same link?

**Review by blva — APPROVED** [review]

File: README.md
Comment (andreaangiolillo) [reply README.md:121]: yes, correct

File: README.md
Comment (andreaangiolillo) [inline README.md:75]: @gssbzn Let me know if this sentence is enough or you want me to add/remove something here

File: README.md
Comment (gssbzn) [inline README.md:109]: these are the same, any reason to have them duplicated?

**Review by gssbzn — CHANGES_REQUESTED** [review]
not sure what your end goal was with this change but if it was to expose Atlas CLI at least for me it's not working as expected.
The README it's becoming too much to parse and too much to scroll if I'm only interested in atlas cli.
Some alternatives to consider.
1. Create a `mongocli.md` and a `atlascli.md` REAMEs and link to them from the main README
2. Delete all content that exists somewhere else and leave only the essential, how to install, login, usage, link all to public docs, requirements about go and install from souce exist in the contributing file can be deleted. Probably also delete both gifs. TL;DR leave the readme a barebone of what the CLIs are link to publci docs for anything else

File: README.md
Comment (fmenezes) [reply README.md:121]: A follow-up here, will the GitHub APIs to list releases work on the previous repo URL?

Comment (fmenezes) [reply README.md:121]: meaning the CLIs out there already (v1.20.0 for instance) will that one be able to verify latest version after repo rename?

File: README.md
Comment (JuliaMongo) [inline README.md:74]: To use `mongocli` with Atlas, open your terminal, run `mongocli auth login`, and follow the prompted steps.

File: README.md
Comment (JuliaMongo) [inline README.md:80]: To use `mongocli`, create API keys. To learn more, see the documentation for the service you're using:

File: README.md
Comment (JuliaMongo) [inline README.md:81]: turn these into a bulleted list.

File: README.md
Comment (JuliaMongo) [inline README.md:87]: Change:
To set up your credentials, run `mongocli config`, or use [env variables] (https://docs.mongodb.com/mongocli/stable/configure/environment-variables/) instead.

File: README.md
Comment (JuliaMongo) [inline README.md:117]: AtlasCLI is currently in the pre-release phase, do not use it in the production environment.

File: README.md
Comment (JuliaMongo) [inline README.md:126]: ease of use

File: README.md
Comment (JuliaMongo) [inline README.md:169]: Here and everywhere, use this structure:
To do X, run

This means, remove "you'll need to"

File: README.md
Comment (JuliaMongo) [inline README.md:122]: After you download the library, you can run it from anywhere and don't need to install it into a global location.

**Review by JuliaMongo — COMMENTED** [review]
LGTM for copy review with these comments.

File: README.md
Comment (fmenezes) [reply README.md:121]: This comment is not meant to block this PR btw

**Review by gssbzn — COMMENTED** [review]
Thanks for the split of files, I think it makes a difference on organising the instructions, left some small comments to address before merging

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: Updated

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: Updated

File: mongocli.md
Comment (andreaangiolillo) [reply mongocli.md]: Updated

File: mongocli.md
Comment (andreaangiolillo) [reply mongocli.md:93]: Updated

**Review by gssbzn — APPROVED** [review]
LGTM thanks for those changes

File: atlascli.md
Comment (dianchenghu) [inline atlascli.md:17]: "It would be best to place this binary somewhere in your `PATH` for ease of use. " sounds more friendly

File: mongocli.md
Comment (dianchenghu) [inline mongocli.md:7]: and automate management …

File: atlascli.md
Comment (dianchenghu) [inline atlascli.md:53]: For consistency, suggest: "or check our [documentation](https://docs.mongodb.com/mongocli/master/) for more details."

File: mongocli.md
Comment (dianchenghu) [inline mongocli.md:63]: For consistency, suggest: "or check our [documentation](https://docs.mongodb.com/mongocli/master/) for more details."

File: atlascli.md
Comment (dianchenghu) [inline atlascli.md]: The product name is "Atlas CLI", not sure if we should add a space here and all the AtlasCLI below.

File: atlascli.md
Comment (dianchenghu) [inline atlascli.md]: The product name is "Atlas CLI", not sure if we should add a space here. I see "MongoDB CLI" below, so I assume we should add a space.

File: README.md
Comment (dianchenghu) [inline README.md:10]: could remove "to". "…, and automate management tasks for your deployments."

**Review by dianchenghu — COMMENTED** [review]
Add some copy suggestions

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md:17]: I will leave this comment open to see what doc team think about this chage

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: updated

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: updated

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: updated

File: README.md
Comment (andreaangiolillo) [reply README.md:10]: updated

**Review by gssbzn — APPROVED** [review]
LGTM

File: atlascli.md
Comment (sarahsimpers) [inline atlascli.md:9]: Nit: 
```suggestion
Atlas CLI is currently in the pre-release phase. Do not use it in a production environment.
```

File: atlascli.md
Comment (sarahsimpers) [reply atlascli.md:17]: I think either is fine, or leave off "Ideally" and just say "You can place this binary somewhere in your `PATH` for ease of use."

**Review by sarahsimpers — DISMISSED** [review]
LGTM % one optional nit for language.

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: updated

File: atlascli.md
Comment (andreaangiolillo) [reply atlascli.md]: Updated

**Review by dianchenghu — APPROVED** [review]
lgtm, thanks!

**Review by blva — APPROVED** [review]

**Review by gssbzn — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]

**Pull request merged**
